src/polymer/js-utils.ts

```diff
@@ -48,15 +47,15 @@ export function toScannedPolymerProperty(
 
   const value = babel.isObjectProperty(node) ? node.value : node;
 
-  let type = closureType(value, sourceRange, document);
-  const typeTag = jsdoc.getTag(parsedJsdoc, 'type');
-  if (typeTag) {
-    type = doctrine.type.stringify(typeTag.type!) || type;
-  }
-  if (type instanceof Warning) {
-    warnings.push(type);
+  const typeResult = getClosureType(value, parsedJsdoc, sourceRange, document);
+  let type;
+  if (typeResult.successful) {
+    type = typeResult.value;
+  } else {
+    warnings.push(typeResult.error);
     type = 'Object';
```

Comment:
Should this be `*` by default? Closure `Object` excludes primitives.

Reply:
If it's a primitive the user should have passed in the primitive constructor, like `String`, `Boolean`, etc